explainer.md

```diff
@@ -0,0 +1,329 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
```

Comment:
Lots of great stuff. I think a couple things that might be helpful would be explaining things that the a11y api explicitly _does not_ attempt to fix and also discussing how this relates to focus. Those two things might be related :) Since I didn't see any mention of focus in here I'm assuming that a11y-api has no effect on it but I'm not sure